src/server/listen.js

```diff
@@ -23,7 +23,8 @@ import http from 'http';
 import https from 'https';
 
 function listenHttp(app, cb) {
-  const port = process.env.HTTP_PORT || '3000';
+  // when `undefined` is passed to process.env it sets it to the string undefined...
+  const port = (process.env.HTTP_PORT !== 'undefined' && process.env.HTTP_PORT) || process.env.PORT;
```

Comment:
Adding this logic because `PORT` is the env var heroku expects apps to use to know where to listen.

Reply:
Took me too long 😓but I'm now realizing that should be done in `defaultValue` of `src/server/config/env/runTime.js`.
```suggestion
  const port = process.env.HTTP_PORT;
```